nobl9/resource_agent_test.go

```diff
@@ -9,6 +9,9 @@ import (
 )
 
 func TestAcc_Nobl9Agent(t *testing.T) {
+	// we dont' currently support creating Agents via the API
+	// so skipping this for now
+	t.SkipNow()
```

Comment:
Our e2e tests are run against main and staging where we can create Agents via the API - maybe we can use main (or staging) as well for tests? In the end SWEs will manage the terraform-provider and we should release changes in our API together with changes in the provider.

I suggest starting with following envs:
```
NOBL9_OKTA_URL=https://accounts.nobl9.dev/
NOBL9_OKTA_AUTH=ausdh5avfxFaHRKHN4x6
NOBL9_URL=http://main.nobl9.dev/api
```

Reply:
> Our e2e tests are run against main and staging where we can create Agents via the API - maybe we can use main (or staging) as well for tests?

Since end users aren't able to create those programmatically, at least in this case we should do the acceptance tests based on how users would use it.  We already had a number of issues related to exposing that functionality, so it's best not to depend on it.

Reply:
Ok, get it. Sounds good for now. When we will be ready to roll terraform provider in parity with API changes we will come back to this topic.